plasma_framework/test/src/exits/payment/PaymentPiggybackInFlightExitOnInput.test.js

```diff
@@ -242,6 +240,17 @@ contract('PaymentInFlightExitRouter', ([_, alice, inputOwner, nonInputOwner, out
             );
         });
 
+        it('should fail when failed to get the block timestamp info of the exit position', async () => {
```

Comment:
Let's name this test and the one in `PaymentPiggybackInFlightExitOnOutput.test.js` the same.
I think we should use different error message, how about:
There is not block for the utxo`? That's basically what is going on here.